content/en/tracing/trace_collection/open_standards/otel_collector_datadog_exporter.md

```diff
@@ -10,319 +10,338 @@ further_reading:
 
-    In the daemonset's `spec.containers.env` section:
+In order to collect valuable Kubernetes Infrastructure metrics, which are used for Datadog Container tagging, we must ensure to report the Pod IP as a resource attribute. To do that, the following snippet is [part of the example][13]:
 
-    ```yaml
-      # ...
-      env:
-         # Get pod ip so that k8sattributes can tag resources
+```yaml
+# ...
+        env:
         - name: POD_IP
           valueFrom:
             fieldRef:
               fieldPath: status.podIP
-          # This is picked up by the resource detector
+        # The k8s.pod.ip is used to associate pods for k8sattributes
         - name: OTEL_RESOURCE_ATTRIBUTES
           value: "k8s.pod.ip=$(POD_IP)"
-      # ...
-    ```
+# ...
+```
 
-    In the `otel-agent-conf` ConfigMap's `data.otel-agent-config` `processors` section:
-
-    ```yaml
-      # ...
-      # The resource detector injects the pod IP
-      # to every metric so that k8sattributes can
-      # fetch information afterwards.
-      resourcedetection:
-        detectors: [env]
-        timeout: 5s
-        override: false
-      # The k8sattributes processor in the Agent is in passthrough mode
-      # so that it only tags with the minimal info for the
-      # collector k8sattributes to complete
-      k8sattributes:
-        passthrough: true
-      # ...
-    ```
+This ensures that [Kubernetes Attributes Processor][14] which is used in [our config map][15] is able to extract the necessary metadata to attach to traces. There are additional [roles][16] that need to be set in order to allow access to this metadata.
 
-    In the `otel-agent-conf` ConfigMap's `data.otel-agent-config` `service.pipelines.traces` section:
+The [example visible in the repository][11] is complete, ready-to-use and has the correct roles set up. All that is required of you is to provide your [application container][17].
 
-    ```yaml
-      # ...
-      # resourcedetection must come before k8sattributes
-      processors: [batch, resourcedetection, k8sattributes]
-      # ...
-    ```
+To learn more about configuring your application, please see the [Application Configuration](#application-configuration) section below.
 
-3. For OpenTelemetry Collectors in standalone collector mode, which receive traces from downstream collectors and export to Datadog's backend, include a `batch` processor and a `k8sattributes` processor. These should be included along with the `datadog` exporter and added to the `traces` pipeline.
+### Gateway Collector Service
 
-    In the `otel-collector-conf` ConfigMap's `data.otel-collector-config` `processors` section:
+For Gateway deployments, we'll first need to set up each [OpenTelemetry Collector agent][18], just like in the [DaemonSet deployment](#daemonset-deployment). Then, the DaemonSet needs to be changed to include an [OTLP exporter][19] instead of the Datadog Exporter [currently in place][20]:
```

Comment:
```suggestion
For Gateway deployments: 

1. Set up each [OpenTelemetry Collector agent][18], just like in the [DaemonSet deployment](#daemonset-deployment). 

2. Change the DaemonSet to include an [OTLP exporter][19] instead of the Datadog Exporter [currently in place][20]:
```